great_expectations/core/usage_statistics/anonymizers/profiler_run_anonymizer.py

```diff
@@ -0,0 +1,205 @@
+import logging
+from numbers import Number
+from typing import Any, Dict, List, Optional, Union
+
+from great_expectations.core.usage_statistics.anonymizers.anonymizer import Anonymizer
+from great_expectations.core.usage_statistics.anonymizers.batch_request_anonymizer import (
+    BatchRequestAnonymizer,
+)
+from great_expectations.util import deep_filter_properties_iterable
+
+logger = logging.getLogger(__name__)
+
+
+class ProfilerRunAnonymizer(Anonymizer):
+    def __init__(self, salt: Optional[str] = None) -> None:
+        super().__init__(salt=salt)
+
+        self._salt = salt
+        self._batch_request_anonymizer = BatchRequestAnonymizer(self._salt)
```

Comment:
Since this is used in the nested builder configs, I figured it made sense to just set a private attr and leverage that - this is a slight deviation from the CheckpointRunAnonymizer.